libs/blocks/hero-marquee/hero-marquee.css

```
@@ -0,0 +1,581 @@
+}
+
+.hero-marquee .m-icon .lockup-area picture img, 
+.hero-marquee.m-icon .lockup-area picture img {
+  min-width:var(--icon-size-m);
+  height: var(--icon-size-m);
+}
+
+.hero-marquee .xl-icon .lockup-area,
+.hero-marquee.xl-icon .lockup-area {
+  font-size: var(--type-body-xl-size);
+  line-height: var(--type-body-xl-lh);
+}
+
+.hero-marquee .xl-icon .lockup-area picture img,
+.hero-marquee.xl-icon .lockup-area picture img {
+  min-width:var(--icon-size-xl);
+  height: var(--icon-size-xl);
+}
+
+.hero-marquee.center {
+  text-align: center;
+  align-items: center;
+}
+
+.hero-marquee.center .action-area,
+.hero-marquee.center .lockup-area { 
+  justify-content: center; 
+}
+
+.hero-marquee .norm p { margin: var(--spacing-xs) 0; }
+.hero-marquee .main-copy p { margin: 0; }
+.hero-marquee .main-copy p:only-child { margin: 0; }
+.hero-marquee .main-copy [class^="heading"] { margin: 0; }
+
+/* Static type */
+.hero-marquee .norm p:only-child { margin: 0; }
+.hero-marquee .norm :is(h1, h2, h3, h4, h5, h6) { margin: 0 0 var(--spacing-xs) 0; }
+.hero-marquee .norm .action-area { margin-top: var(--spacing-s); }
+.hero-marquee .norm div > *:last-child { margin-bottom: 0; }
+.hero-marquee .norm div *:first-child { margin-top: 0; }
+
+.hero-marquee > .foreground {
+  max-width: var(--grid-container-width);
+  min-width: var(--grid-container-width);
+  margin: 0 auto;
+  display: grid;
+  gap: var(--spacing-m);
+}
+
+html[dir="rtl"] .hero-marquee .foreground {
+  flex-direction: row-reverse;
+}
+
+.hero-marquee > .foreground.fw {
+  width: 100%;
+}
+
+.hero-marquee .foreground .copy {
+  display: grid;
+  gap: var(--spacing-xs);
+}
+
+.hero-marquee hr {
+  width: 100%;
+  border: none;
+  height: 1px;
+  margin: 0;
+}
+
+.hero-marquee .has-divider {
+  margin: var(--spacing-xxs) 0;
+}
+
+.hero-marquee .badge {
+  font-size: var(--type-body-xs-size);
+  line-height: 1em;
+  font-weight: 400;
+  vertical-align: middle;
+  padding: 4px 8px;
+  margin-inline-start: var(--spacing-xs);
+  border-radius: 4px;
+  background-color: #FFEB00;
+  color: var(--text-color);
+  white-space: no-wrap;
+}
+
+/* content hidden */
+.hero-marquee .hidden {
+  visibility: hidden;
+  height: 0;
+}
+
+.hero-marquee .foreground div:empty {
+  display: none;
+}
+
+/* Row Types */
+.hero-marquee .row-list {
+  text-align: left;
+} 
+
+html[dir="rtl"] .hero-marquee .row-list {
+  text-align: right;
+}
+
+.hero-marquee .row-list .row-wrapper {
+  display: table;
+}
+
+.hero-marquee .row-list li {
+  margin-bottom: var(--spacing-xxs);
+} 
+
+/* row-qr */
+.hero-marquee .row-qrcode .row-wrapper {
+  display: flex;
+  gap: var(--spacing-s);
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .row-wrapper > p {
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .qr-code-img {
+  display: none;
+}
+
+.hero-marquee .row-qrcode .google-play a,
+.hero-marquee .row-qrcode .app-store a {
+  width: 135px;
+  height: 40px;
+  display: inline-flex;
+  text-indent:-999em;
```

I have mixed feeling about this strategy of hiding the text. Could we set the color to transparent or remove the text in JS?
```suggestion
  text-indent: -999em;
```

---

Good thought, I'm a little weary of this strategy as well. 
Since this is also used for `daa-ll` attributes, I've just set the `color: transparent`.